prow/Makefile

```diff
@@ -59,11 +58,6 @@ hook-deployment:
 hook-service:
 	kubectl create -f cluster/hook_service.yaml
 
-line-image:
-	CGO_ENABLED=0 go build -o cmd/line/line k8s.io/test-infra/prow/cmd/line
-	docker build -t "gcr.io/$(PROJECT)/line:$(LINE_VERSION)" cmd/line
-	gcloud docker -- push "gcr.io/$(PROJECT)/line:$(LINE_VERSION)"
```

Comment:
I was about to say .PHONY but it's not there...

I guess it was `test-pr-image`